e2e/playwright/shared/api/api-client-factory.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2005 - 2023 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+
+import {
+  AdminTenantsApi,
+  AdminUsersApi,
+  AlfrescoApi, ContentClient,
+  GroupsApi,
+  NodesApi,
+  PeopleApi,
+  QueriesApi,
+  SearchApi,
+  SecurityGroupsApi, SecurityMarksApi,
+  SitesApi,
+  UploadApi
+} from '@alfresco/js-api';
+
+import { users } from '../page-objects/global-variables';
+
+import {logger} from "@alfresco/adf-cli/scripts/logger";
```

Comment:
incorrect brackets format, should be single quotes for imports, same for the line below